recipes/napari-DeepSpot/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "napari-DeepSpot" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-DeepSpot-{{ version }}.tar.gz
+  sha256: 963c550cdea4da2bbb3b2bc865d1c3f66baf826d43afd5272ba2470febb8ebe0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools-scm
+  run:
+    - napari
+    - napari-plugin-engine >=0.1.4
+    - numpy
+    - opencv
+    - pyqt
+    - pytest
+    - pytest-cov
+    - pytest-qt
+    - pytest-xvfb
+    - python >=3.7
+    - qtpy ==1.9.0
```

Comment:
This needs to be relaxed a bit since a lot of tools depend on it in the napari ecosystem.

Reply:
I don't think this specific version is needed, qtpy is only used for QtWidgets.